packages/dev/core/src/Engines/engine.ts

```diff
@@ -1242,18 +1242,22 @@ export class Engine extends ThinEngine {
     }
 
     protected _cancelFrame() {
-        if (this._renderingQueueLaunched && this.customAnimationFrameRequester) {
-            this._renderingQueueLaunched = false;
-            const { cancelAnimationFrame } = this.customAnimationFrameRequester;
-            if (cancelAnimationFrame) {
-                cancelAnimationFrame(this.customAnimationFrameRequester.requestID);
+        if (this.customAnimationFrameRequester) {
+            if (this._frameHandler > -1) {

```

Comment:
```suggestion
            if (this._frameHandler !== 0) {
```
The [requestAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) doc says that the returned value is a non-zero long value and that no other assumption should be made.